admin_ui/backend/agents_store.py

```
@@ -0,0 +1,160 @@
+
+    def __init__(self, db_path: str = DB_DEFAULT):
+        parent = os.path.dirname(db_path)
+        if parent:
+            os.makedirs(parent, exist_ok=True)
+        self.conn = sqlite3.connect(db_path)
+        self.conn.row_factory = sqlite3.Row
+        self.conn.execute("PRAGMA journal_mode=WAL")
+        self.conn.execute("PRAGMA busy_timeout=5000")
+        self.conn.execute("PRAGMA synchronous=NORMAL")
+        self.conn.executescript(SCHEMA)
+        try: os.chmod(db_path, 0o600)
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)